canonical-websites/www.ubuntu.com
# Beta release

## Done

Rebase `master` onto `beta-1.7.0`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check that nothing looks broken


## Issue / Card

Fixes #3337
